src/templates/services/index.hbs

```diff
@@ -53,7 +53,7 @@
         <div>
           <img src="/assets/img/services/between-45-55.svg" width="355" height="355" alt="" />
           <div>
-            <a href="/contact" class="button big">Contact our experts</a>
+            <a href="/contact" class="button big">Talk to a QROPS specialist</a>
```

Comment:
The fuck is a QROPS. It's a mistake of Finsec to put a weird word here that no user will understand. Simply "talk to our expert" makes sense to everyone.